core/src/main/java/org/apache/accumulo/core/clientImpl/mapreduce/lib/InputConfigurator.java

```diff
@@ -230,7 +231,7 @@ public static void setRanges(Class<?> implementingClass, Configuration conf,
       conf.setStrings(enumToConfKey(implementingClass, ScanOpts.RANGES),
           rangeStrings.toArray(new String[0]));
     } catch (IOException ex) {
-      throw new IllegalArgumentException("Unable to encode ranges to Base64", ex);
+      throw new UncheckedIOException("Unable to encode ranges to Base64", ex);
```

Comment:
This one actually is an illegal argument. This is user-facing API, and the IOException is just incidental to the implementation. The user-facing exception should be IAE.